spec/system/user_logs_in_with_password_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe "Authenticating with a password" do
   def submit_login_form(email, password)
+    allow(ForemInstance).to receive(:smtp_enabled?).and_return(true)
```

Comment:
this fixes a failing spec (not sure why it passed in the PR?) - oddly this works when run in isolation and may be tied to interaction (see the clear_cache change earlier).